frontend/src/pages/projects/suite.js

```diff
@@ -0,0 +1,23 @@
+import { create, test, enforce, only } from "vest";
+
+const suite = create((data = {}, fieldName) => {
+    only(fieldName);
+
+    test("short_title", "This is required information", () => {
+        enforce(data.short_title).isNotBlank();
+    });
+    test("short_title", "Nickname must be 3 letters", () => {
```

Comment:
made this up, not sure if needed

Reply:
Seems valid, might want to reflect it in a .feature rule though.